src/qpm.io/qpm/commands/init.go

```diff
@@ -115,6 +118,20 @@ func (ic *InitCommand) Run() error {
 	if len(filename) == 0 {
 		filename = ic.Pkg.PriFile()
 	}
+
+	re := regexp.MustCompile("[-\\.]")
```

Comment:
I suggest moving this to the top of this file and making it a package global variable like so:

```
var regexGitHubLicense = regexp.MustCompile("[-\\.]")
```

The reason for doing this is that package globals are initialized when the package is loaded so if there are errors in the regexp it will fail immediately instead of later when the code executes. This also ensures the regexp is only compiled once instead of every time this code executes. Not a huge issue since this regexp is only used once, but it's a good pattern to follow :)